### PR TITLE
proposed change

### PR DESCRIPTION
**Pull request by rahulsust**

### CODE REVIEW

File: archive/readme.md [hunk 1]
```diff
@@ -38,8 +38,7 @@ Then add the cards into it.
 
 ## How does it work
 
-The `getArchiveCards()` method creates `<object>` elements. As many as there are archive files (based on the manually updated `NUMBER_OF_FILES`).  
-Each object element imports the archive file with the corresponding index.  
+The getArchiveCards() method creates an <object> element for each archived file. The total number of archived files is based on the manually updated NUMBER_OF_FILES.
```
Comment (lax0248917) [inline archive/readme.md:41]: Hi @rahulsust, this has been a really good learning process for us both on code reviews 😆 . Before I can merge this it appears the backticks (`) were removed from getArchiveCards(), the object tag and the NUMBER_OF_FILES variable. Its important to add these in markdown files to have code snippets render correctly. If you could add those backticks back in that would be awesome!! Thank you 👍

**Review by lax0248917 — CHANGES_REQUESTED** [review]

File: archive/readme.md (hunk 1 above)
Comment (rahulsust) [reply archive/readme.md]: Hey @lax0248917, I overlooked those mistakes 😅.  I am enjoying working with you.  Thanks for your continuous support.

File: archive/readme.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Then add the cards into it.
 
 ## How does it work
 
-The getArchiveCards() method creates an <object> element for each archived file. The total number of archived files is based on the manually updated NUMBER_OF_FILES.
+The 'getArchiveCards()' method creates an '<object>' element for each archived file. The total number of archived files is based on the manually updated 'NUMBER_OF_FILES'.
```
Comment (lax0248917) [inline archive/readme.md:41]: Hello @rahulsust looks like you used single quotes (' '), you want to use backticks found above the Tab key on a US Keyboard. Markdown interprets the backticks as a code snippet, rendering them like this as an example: `your code here`. Let me know if you have any questions and appreciate you taking care of this 🚀

**Review by lax0248917 — CHANGES_REQUESTED** [review]

**Review by lax0248917 — APPROVED** [review]
LGTM!!!

**Pull request merged**